ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -197,6 +198,8 @@ public Response handleKsqlStatements(
       @Context final ServiceContext serviceContext,
       final KsqlRequest request
   ) {
+    // Set masked sql statement if request is not from OldApiUtils.handleOldApiRequest
```

Comment:
Same comment as in the 5.5.x backport about removing this comment because OldApiUtils no longer exists. We can switch the below from `setMaskedSqlIfNeeded()` to `setMaskedSql()` as well.

Reply:
I might missed it. Why don't we use `setMaskedSqlIfNeeded` as it's safer?

Reply:
Never mind, I misunderstood what you meant.